dashboard/app/views/pd/workshop_mailer/teacher_follow_up.html.haml

```diff
@@ -0,0 +1,57 @@
+:css
+  body {
+    font-family: Gotham, sans-serif;
+  }
+
+- if @workshop.subject == Pd::Workshop::SUBJECT_CSF_101
+  %p
+  Hi
+  = @enrollment.first_name + ','
+
+  %p
+  It has been one month since your CS Fundamentals Intro workshop. What steps have you taken since
+  then to bring CS Fundamentals to your school or classroom? From collaborating with colleagues,
+  to setting up class sections in the dashboard, to scheduling lessons, there are lots of ways to
+  ensure computer science education is accessible to all students. We would love to hear how it is
+  going! Tweet us stories and pictures @TeachCode!
+
+  %p
+  If you haven’t used CS Fundamentals in your classroom yet, that’s okay! Here are some steps to
+  help you get started:
+
+  %h3 Step 1: Choose a course
+  On Code.org, you’ll find
+  = link_to('courses for all grade levels', 'https://studio.code.org/courses?view=teacher')
+  from pre-readers to seniors in high school, and everyone in between. All our courses are available
+  at no cost. Chat with colleagues at your school to determine how the course curriculum can be
+  integrated with a variety of subjects.
+
+  %h3 Step 2: Set up your classroom section
+  = link_to 'Set up a classroom section', 'https://studio.code.org/home'
+  to view your student’s progress, print login cards for your students, manage their accounts,
+  and print certificates they can bring home when they finish the course.
+
+  %h3 Step 3: Start teaching
+  All it takes is one lesson to bring computer science concepts into your classroom. Once you see
+  your students enthusiasm for CS, go further with daily lesson plans to guide your students,
+  = link_to 'unplugged activities', 'https://code.org/curriculum/unplugged'
+  you can do without a computer, and
+  = link_to 'forums', 'https://forum.code.org/'
+  to connect to other teachers.
+
+  %h3
+    = link_to 'Get started here', 'https://studio.code.org/home'
+
+  %p
+  Remember that you can always reach out to us for support at
+  = mail_to('teacher@code.org') + '.'
+
+  %p
+    Thank you,
+    %br Megan Hochstatter
+    %br K-5 Education Program Manager, Code.org
+
+- else
```

Comment:
Would this make more sense as a comment at the top of the file? The only way you would see this `else` get rendered is either if we accidentally sent this email, in which case the recipient would see it, not us, or if you are looking at a preview, in which case you would see all the csf-specific content anyway. I'm not opposed to including this, I'm just not sure what scenario we are targeting.

Reply:
I'll change it to be a Ruby comment